UniversalAlgebra/Varieties/EquationalLogic.lagda

```diff
@@ -257,8 +260,8 @@ module Completeness {χ ι : Level}{I : Type ι} (E : I → Eq{χ}) {Γ} where
  completeness p q V = begin
                   p              ≈˘⟨ identity p ⟩
                   p [ σ₀ ]       ≈˘⟨ evaluation p σ₀ ⟩
-                  apply ⦅ p ⦆ σ₀  ≈⟨ V (M Γ) satisfies σ₀ ⟩
-                  apply ⦅ q ⦆ σ₀  ≈⟨ evaluation q σ₀ ⟩
+                  ⦅ p ⦆ <$> σ₀  ≈⟨ V (M Γ) satisfies σ₀ ⟩
```

Comment:
As `<$>` is always applied to something that looks like ` ⦅ _ ⦆`, you might want to make `<$>` build-in ` ⦅ _ ⦆` which will lighten the presentation.